resolwe/process/runtime.py

```diff
@@ -8,6 +9,16 @@
 
 try:
     from plumbum import local as Cmd  # pylint: disable=unused-import
+    # Log plumbum commands to standard output.
+    FORMATTER = logging.Formatter('%(asctime)s - %(levelname)s - %(name)s[%(process)s]: %(message)s')
```

Comment:
These are not constants, so you should keep them lowercase - and disable pylint.